examples/CIB/ex1/example.cpp

```diff
@@ -327,13 +329,78 @@ run_example(int argc, char* argv[])
         }
         navier_stokes_integrator->setStokesSolverNeedsInit();
 
+        // Set up the hydro force objects
+        double rho_fluid = input_db->getDouble("RHO");
+        double mu_fluid = input_db->getDouble("MU");
+        Pointer<IBHydrodynamicForceEvaluator> hydro_force =
+            new IBHydrodynamicForceEvaluator("IBHydrodynamicForce", rho_fluid, mu_fluid, true);
+
+        // Get the initial box position and velocity from input
+        const string init_hydro_force_box_out_db_name = "InitHydroForceBox_0";
+        SAMRAI::tbox::Array<double> box_X_lower_array_out, box_X_upper_array_out, box_init_vel_array_out;
+        IBTK::Vector3d box_X_lower_out, box_X_upper_out, box_init_vel_out;
+
+        box_X_lower_array_out =
+            input_db->getDatabase(init_hydro_force_box_out_db_name)->getDoubleArray("lower_left_corner");
+        box_X_upper_array_out =
+            input_db->getDatabase(init_hydro_force_box_out_db_name)->getDoubleArray("upper_right_corner");
+        box_init_vel_array_out =
+            input_db->getDatabase(init_hydro_force_box_out_db_name)->getDoubleArray("init_velocity");
+        for (int d = 0; d < 3; ++d)
```

Comment:
Instead of the following code, you can do:

`getDoubleArray(key, &array[0], 3)`

This lets you avoid copying between Array<double> and Vector3d.